onnxruntime/core/mlas/lib/quantize.cpp

```diff
@@ -18,7 +18,8 @@ Module Name:
 
 --*/
 
-#include "mlasi.h"
+#include "quantize.h"
```

Comment:
This file shouldn't need to change now, right?

Reply:
Yes.